plugin/coc-quick-lint-js/src/index.ts

```diff
@@ -0,0 +1,122 @@
+import * as cp from 'child_process';
```

Comment:
Almost-blocking: Is there a reason why we're not reusing code from the VS Code plugin? The APIs this plugin is interacting with look similar to the APIs our existing VS Code extension uses.

Reply:
Hi, I'm using `quick-lint-js-wasm` like VSCode, but came error `TextEncoder is not defined`.

Reply:
I've tested with Node.js 14 and 16, both are failed.

Reply:
> I've tested with Node.js 14 and 16, both are failed.

Hmm, weird. Perhaps we can find a workaround. How can I test it on my machine?

Reply:
First, you need to install coc.nvim by something like `Plug 'neoclide/coc.nvim', {'branch': 'release'}`, then add `set runtimepath^=~/src/quick-lint-js/plugin/coc-quick-lint-js` to your vimrc.

1. `cd wasm` and build https://github.com/quick-lint/quick-lint-js/tree/master/wasm
1. `cd plugin/coc-quick-lint-js`
2. `yarn install`
3. `vim xxx.js`, `:CocList extensions` will list `coc-quick-lint-js`.

Reply:
I can make it working by add `const { TextDecoder, TextEncoder } = require("util");` to `wasm/quick-lint-js.js`.